<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="main_layout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="attrWork">
         <attribute name="title">
          <string>Атрибуты</string>
         </attribute>
         <widget class="QListView" name="attrList">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>381</width>
            <height>428</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="addNewAttrBtn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>440</y>
            <width>381</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="delAttrBtn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>470</y>
            <width>381</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="styleWork">
         <attribute name="title">
          <string>Стили</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListView" name="styleList"/>
          </item>
          <item>
           <widget class="QPushButton" name="addNewStlBtn">
            <property name="text">
             <string>Добавить</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delStlBtn">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="dom_layout">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <layout class="QFormLayout" name="dom_control_layout">
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Добавление узла</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="insertTypeLable">
            <property name="text">
             <string>Тип</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="insertTypeComboBox"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="insertInDOMBtn">
            <property name="text">
             <string>Добавить элемент</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="removeFromDOMBtn">
            <property name="text">
             <string>Удалить элемент</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string> Файл</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
